docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.11.5
```

Comment:
I think I mentioned this in PT, but the reason why I used Alpine is so that I could use `/bin/sh`. It saves me from trying to update `geoipupdate` without breaking the library. It only adds 2.5 MB or so.